tests/phpunit/tests/tm/test-wpml-pb-handle-custom-fields.php

```diff
@@ -253,6 +253,19 @@ public function it_does_not_slash_non_json() {
 		$not_json = 'do not escape these - \\ " \'';
 		$this->assertEquals( $not_json, WPML_PB_Handle_Custom_Fields::slash_json( $not_json ) );
 	}
+
+	/**
+	 * @test
+	 * @group wpmlcore-7141
+	 */
+	public function it_does_not_slash_is_original_data_is_not_a_string() {
```

Comment:
```suggestion
	public function it_does_NOT_slash_if_original_data_is_NOT_a_string() {
```

@strategio:

- Typo: "is" -> "if"
- Capitalized negation to make them more prominent when looking at the test names